test/Dialect/FIRRTL/errors.mlir

```diff
@@ -24,8 +24,7 @@ firrtl.module @X(in %b : !firrtl.uint<32>, in %d : !firrtl.uint<16>, in %out : !
 // expected-error @+1 {{'firrtl.circuit' op must contain one module that matches main name 'MyCircuit'}}
 firrtl.circuit "MyCircuit" {
 
-"firrtl.module"() ( {
-}) { type = () -> ()} : () -> ()
+firrtl.module @X() {}
```

Comment:
FWIW, this was changed so the desired error is triggered, since now the module is rejected first with:

```
error: unexpected error: 'firrtl.module' op region #0 ('body') failed to verify constraint: region with 1 blocks
```